source/control/server_config.cc

```diff
@@ -29,16 +29,16 @@ namespace psyllid
     {
         // default server configuration
 
-        param_node* t_amqp_node = new param_node();
-        t_amqp_node->add( "broker", new param_value( "localhost" ) );
-        t_amqp_node->add( "queue", new param_value( "psyllid" ) );
-        t_amqp_node->add( "slack-queue", new param_value( "slack_interface" ) );
+        param_node t_amqp_node = param_node();
+        t_amqp_node.add( "broker", param_value( "localhost" ) );
```

Comment:
I like this, but I think it would need lots of overloads (maybe it could be templated?) to be a reasonable interface; I'd think we want every constructor available in param_value.

Reply:
We can do it with a single overload because of implicit conversion between bool/uint/int/double/string and param_value.  I will test this out and make sure it works.

Reply:
can this comment mark the requested change as done